assets/stylesheets/app/_flight_manual.scss

```diff
@@ -481,6 +481,17 @@ input.documents-search-input {
   }
 }
 
+// Adjustments for page-specific links to improve the docs
+$improve-link-separator-spacing: 1em;
```

Comment:
@simurai Forgot to include a preview! 

<img width="744" alt="napkin 2 03-25-16 2 29 41 pm" src="https://cloud.githubusercontent.com/assets/330036/14050977/03b39566-f296-11e5-83b2-9b9521440405.png">

We can, of course, easily add that pipe separator back if we want to :grin: